llvm/tools/llvm-jitlink/llvm-jitlink-executor/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(LLVM_LINK_COMPONENTS
   OrcShared
   OrcTargetProcess
   Support
+  Core
```

Comment:
Why did you need to add link dependencies?

Reply:
Good question! @Bigcheese and myself discussed. We technically may not need `Core` here and I will revise. The dependency comes from https://github.com/llvm/llvm-project/blob/00e1cc4c9d002c78cf890b630343b052ebca0399/llvm/include/llvm/SandboxIR/Type.h#L286. Specifically, the new `SandBoxIR` code uses a function (`Type::print`) from `LLVMCore`.  Linking fails without the dependency.

The `SandBoxIR` code currently does not have module maps at all and I will go ahead and add them. Maybe we will no longer need the dependencies after the module maps are added.

Reply:
Code is revised to avoid adding the Core dependency.